### ISSUE TITLE
Error while running the ec2 terraform script. 

### ISSUE THREAD

**Issue by ghostinroot** [RhinoSecurityLabs/cloudgoat#24]
./cloudgoat.py create rce_web_app 
Using default profile "cloudgoat" from config.yml...
Loading whitelist.txt...
A whitelist.txt file was found that contains at least one valid IP address or range.

Now running rce_web_app's start.sh...
There are some problems with the configuration, described below.

The Terraform configuration must be valid before initialization so that
Terraform can determine which modules and providers need to be installed.

Error: Error parsing /Users/<****>/cloudgoat/rce_web_app_cgirwmfal/terraform/ec2.tf: At 67:21: Unknown token: 67:21 IDENT var.cg_whitelist

[cloudgoat] Error while running `terraform init`.
    exit code: 1
    stdout: None
    stderr: None

**Comment by rhino-xander** [comment]
@ghostinroot hey - thanks for the bug report!

Unfortunately I'm having trouble replicating the issue on my side - are you able to share some more details on your system?

In particular, it'd be awesome if you could confirm your OS (type and version), the full version numbers of Terraform and python you have installed, and the contents of your whitelist.txt (redact the IP but leave CIDR range).

Cheers!

**Comment by ghostinroot** [comment]
You're exactly right!! 

I did an upgrade to my terraform to 0.12 but didn't link it, brew link terraform did that for me. Thank you.

**Comment by rhino-xander** [comment]
So glad to hear that helped! I've made a note to look at ways to make CloudGoat more aware of what version of Terraform is being used.

**Comment by jensen-ackles** [comment]
I installed terraform in my ubuntu system on aws but it is repeating the same error
Terraform initialized in an empty directory
The directory has no Terraform configuration files. You may begin working
with Terraform immediately by creating Terraform configuration files.

**Comment by rhino-xander** [comment]
Hey @jensen-ackles - sorry to hear you're having trouble, but thanks for giving CloudGoat a try!

Can you please provide some screenshots or direct-copied commandline output for the following?

1. The version of terraform you are using
2. The exact command you ran which resulted in that error
3. A `ls -lah` of your cloudgoat install directory

Thanks!

**Comment by jensen-ackles** [comment]
![Screenshot (18)](https://user-images.githubusercontent.com/57521093/68566983-e2294500-047d-11ea-9cc0-68764f5146df.png)


i am getting this error while trying deploying cloudgoat
if you can please provide solution

**Comment by bbatmandakh** [comment]
I'm getting same error.
Cloudgoat 2.0
Terraform_0.12.16(terraform_0.12.16_linux_amd64.zip)

![cloudgoat_error](https://user-images.githubusercontent.com/24693946/69779470-7835cd00-11eb-11ea-96df-597df9bcc0b0.PNG)


Any solution?

**Comment by rhino-xander** [comment]
@bbatmandakh @jensen-ackles @ghostinroot - can any of you review this solution ( https://github.com/RhinoSecurityLabs/cloudgoat/issues/26#issuecomment-524100877 ) and let me know if that was the issue?

TLDR: AWS creds live in `~/` which is per user, so if you run the AWS configs and cloudgoat under different users (e.g. root and your regular user) the error you encounter will occur.

**Comment by bbatmandakh** [comment]
its working!
